Emby.Server.Implementations/Library/Resolvers/PhotoResolver.cs

```diff
@@ -103,8 +101,8 @@ internal static bool IsImageFile(string path, IImageProcessor imageProcessor)
         {
             ArgumentNullException.ThrowIfNull(path);
 
-            var extension = Path.GetExtension(path.AsSpan()).TrimStart('.');
-            if (!imageProcessor.SupportedInputFormats.Contains(extension, StringComparison.OrdinalIgnoreCase))
+            var extension = Path.GetExtension(path.AsSpan()).TrimStart('.').ToString();
```

Comment:
Unfortunately, hash sets don't have an overload for spans, so we have to allocate a string

Reply:
Actually, I would expose `bool IsInputFormatSupported(string inputFormat)` method instead of `IReadOnlyCollection<string> SupportedInputFormats { get; }` that is a set under the hood.
So we don't have to think about what is under the hood and use what Contains overload should we use.